ownCloudAppShared/Client/File Lists/QueryFileListTableViewController.swift

```diff
@@ -66,6 +66,8 @@ open class QueryFileListTableViewController: FileListTableViewController, SortBa
 	public var duplicateMultipleBarButtonItem: UIBarButtonItem?
 	public var copyMultipleBarButtonItem: UIBarButtonItem?
 	public var openMultipleBarButtonItem: UIBarButtonItem?
+	public var isFolderSelectionOnlyMode: Bool = false
```

Comment:
@hosy hm.. by reading 'isFolderSelectionOnlyMode' I don't really understand what it means...

Reply:
variable was renamed